LTSupportAutomotive/LTOBD2PID.m

```diff
@@ -3,6 +3,7 @@
 //
 
 #import "LTOBD2PID.h"
+#import "float.h"
```

Comment:
If that's just for the `(double)` case, we better `#import <Foundation/Foundation.h>` here.

In Objective-C-land we usually try to avoid the plain C headers like `stdio.h`, and friends.